layers/gd/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION as ext
+
+# Install required build libraries
+RUN set -ue \
+    ; LD_LIBRARY_PATH= \
+    ; yum -y install \
+        libwebp-devel \
+        libXpm-devel \
+        libpng-devel \
+        libjpeg-devel \
+        freetype-devel \
+    ; yum clean all \
+;
+
+# Compile the GD extension
+WORKDIR ${PHP_BUILD_DIR}/ext/gd
+RUN phpize
+RUN ./configure \
+        --disable-static \
+        --enable-gd \
+        --enable-gd-jis-conv \
+        --enable-shared \
+        --with-freetype \
+        --with-jpeg \
+        --with-webp \
+        --with-xpm \
```

Comment:
Any idea if multiple PHP versions are correctly supported? I'm asking because I remember a change in compilation parameters in 7.4: https://github.com/php/php-src/blob/PHP-7.4/UPGRADING#L761-L770

`--with-gd` became `--enable-gd` and `with-png-dir` was removed, which is good here, but what about PHP 7.3?

Reply:
For reference in the 7.3 build file: https://github.com/brefphp/bref/blob/ffdf7378818d579f6b8748214735a02a060f620a/runtime/base/php-73.Dockerfile#L81-L83

Reply:
Good catch!

Only php74 was built in travis, I've recently pushed to build ontop of other versions and it's green but yes, without the png configure flag.

So I will test my snippet code with a png file accross all php versions and make necessary fixes regarding flags if needed.